lib/ansible/modules/cloud/smartos/vmadm.py

```diff
@@ -78,6 +78,11 @@
     required: false
     description:
       - Domain value for C(/etc/hosts).
+  docker:
+    required: false
+    description:
+      - Docker images need this flag enabled along with the brand set to lx.
```

Comment:
Could use please use proper formatting for `brand` and `lx` to make it clear these refer to parameters? e.g. `I(brand) set to C(lx)`.